### PR TITLE
CP-30756: Update base64 and remove stdext.base64

### PR DESCRIPTION
**Pull request by psafont**
Related PRs:
https://github.com/xapi-project/stdext/pull/37
https://github.com/xapi-project/xen-api-libs-transitional/pull/63
https://github.com/xapi-project/xen-api/pull/3832
https://github.com/xapi-project/wsproxy/pull/20

### CODE REVIEW

**Comment by psafont** [comment]
I still have to produce a PR for https://github.com/xapi-project/wsproxy and after that there needs to thorough testing; @mseri said that removing the sanitization from base64 decoding was potentially disruptive but it is not known right now what users of base64 are affected.

**Comment by mseri** [comment]
If I remember correctly the unsanitized base64 was succeeding BVTs, randomly failing BSTs, but was failing often in nightlies. I think we have fixed most, if not all, the few bits that were causing this. You may still be able to find the test logs from when I was testing the new cleaned up stdexts in xenrt.

TL;DR: I am sorry, I cannot really remember the details of the fixes and failures, but if it passes a nightly it should be fine

**Comment by psafont** [comment]
The branch has passed BVTs and BSts, as well as upgrade tests in the nightlies. Looks like most of the work to prevent breaking was done.

All the branches have been updated and are ready to reviewed and merged
